ilastik/shell/gui/reportIssueDialog.py

```diff
@@ -0,0 +1,94 @@
+import os
+
+import ilastik
+import platform
+import sys
+import webbrowser
+from PyQt5.QtWidgets import QApplication, QDialog, QVBoxLayout, QLabel, QTextEdit, QPushButton, QHBoxLayout, QSizePolicy
+from urllib.parse import quote
+
+
+def _get_log_since_last_startup() -> str:
+    """
+    Reads the log up to maximum 1 MB from the end and returns the text since the last startup, or the entire 1 MB.
+    We don't want people emailing us more than 1MB of log...
+    """
+    log_path = ilastik.ilastik_logging.default_config.get_logfile_path()
+    marker = b"Starting ilastik"
+    text_size_limit = 1048576  # 1 MB
+    file_size = int(os.path.getsize(log_path))
+    with open(log_path, "rb") as log_file:
+        position = max(0, file_size - text_size_limit)
+        log_file.seek(position)
+        log_text = log_file.read()
+    if marker in log_text:
```

Comment:
Always returning `b"".join(log_text.rpartition(marker)[1:])` would produce the same results of branching on `marker in logtext`